templates/repo/issue/view_content/pull.tmpl

```diff
@@ -109,8 +109,8 @@
 	</div>
 {{end}}
 <div class="timeline-item comment merge box">
-	<a class="timeline-avatar text  {{if .Issue.PullRequest.HasMerged}}purple
-	{{- else if .Issue.IsClosed}}grey
+	<a class="timeline-avatar text {{if .Issue.PullRequest.HasMerged}}purple
+	{{- else if .Issue.IsClosed}}red
```

Comment:
Why it's red? `Closed` (for issue/PR) is a normal state, it shouldn't get too much attention.

Reply:
It matches the PR state color above. Also GitHub colors closed PR action icon in red as well as a timeline icon.

Reply:
As I see it, PR can be in green, purple and red state and the merge box displays in all three. Only in red state, the color did not match.

Reply:
I guess I will revert, even thought I don't like the inconsistency in colors. GH also colors a closed PR as red, but they only have the timeline event, not the box we have.

<img width="395" alt="Screenshot 2023-04-22 at 21 49 18" src="https://user-images.githubusercontent.com/115237/233803763-d5f80125-1e25-437e-a920-8c704df4d598.png">

Reply:
reverted